Makefile

```diff
@@ -734,7 +734,11 @@ OPTIMIZE    = -O0
 LTO_FLAGS   = $(OPTIMIZE)
 else
 OPTIMIZE    = -Os
+ifeq ($(TARGET),$(filter $(TARGET),$(F1_TARGETS)))
```

Comment:
Believe me its needed! DMA is a mess now with it! Spent like daaays debugging some DMA problems introduced with DSHOT and it was this optimiser screwing it totally up, while in DEBUG=GDB everything was working fine.
Probably the issue is deeper in the code where it doesnt comply for this optimiser, but its easier to just enable it for targets with more space